pkg/rtc/participant.go

```diff
@@ -76,12 +74,10 @@ type ParticipantImpl struct {
 	// hold reference for MediaTrack
 	twcc *twcc.Responder
 
+	uptrackManager *UptrackManager
```

Comment:
Refactoring all up track management into this new module.

Reply:
i noticed we are having to do quite a bit of method forwarders into the uptrackManager. FWIW, if desired, you could include upstrackManager as a composite. i.e.

```
type ParticipantImpl struct {
    UptrackManager
}
```
though doing so will expose all of the public methods on ParticipantImpl